skbio/io/format/tests/test_embl.py

```diff
@@ -0,0 +1,773 @@
+             'comp': ''
+            },
+            {
+             'local_span': '427-526',
+             'primary_identifier': 'AC001234.2',
+             'primary_span': '1-100',
+             'comp': 'c'
+            },
+            {
+             'local_span': '527-1000',
+             'primary_identifier': 'TI55475028',
+             'primary_span': 'not_available',
+             'comp': ''
+            }
+        ]
+
+        # read reference
+        obs = _parse_assembly(lines)
+
+        # See all differences
+        self.maxDiff = None
+        self.assertEqual(obs, exp)
+
+    def test_embl_to_generator_single(self):
+        # test single record and uppercase sequence
+        for c in [Sequence, DNA]:
+            obs = next(_embl_to_generator(
+                self.single_upper_fp, constructor=c))
+            exp = c(self.single[0], metadata=self.single[1],
+                    positional_metadata=self.single[2], lowercase=True)
+            self.assertEqual(exp, obs)
+
+    def test_get_embl_section(self):
+        """Verify to have a section for each embl ID"""
+
+        with open(self.single_rna_fp) as fh:
+            for line in fh:
+                if line.startswith("//"):
+                    continue
+
+                # test that this function doesn't raise exceptions
+                try:
+                    _get_embl_section(line)
+
+                except KeyError as err:
+                    raise EMBLFormatError("Key {0} isn't defined in embl."
+                                          "KEYS_2_SECTIONS".format(err))
+
+    def test_embl_to_generator(self):
+        for i, obs in enumerate(_embl_to_generator(self.multi_fp)):
+            seq, md, imd, constructor = self.multi[i]
+            exp = constructor(seq, metadata=md, lowercase=True,
+                              interval_metadata=imd)
+            self.assertEqual(exp, obs)
+
+    def test_embl_to_sequence(self):
+        for i, exp in enumerate(self.multi):
+            obs = _embl_to_sequence(self.multi_fp, seq_num=i+1)
+            exp = Sequence(exp[0], metadata=exp[1], lowercase=True,
+                           interval_metadata=exp[2])
+            self.assertEqual(exp, obs)
+
+    def test_embl_to_rna(self):
+        seq, md, imd, constructor = self.single_rna
+        obs = _embl_to_rna(self.single_rna_fp)
+        exp = constructor(seq, metadata=md,
+                          lowercase=True, interval_metadata=imd)
+
+        self.assertEqual(exp, obs)
+
+    def test_embl_to_dna(self):
+        i = 1
+        exp = self.multi[i]
+        obs = _embl_to_dna(self.multi_fp, seq_num=i+1)
+        exp = DNA(exp[0], metadata=exp[1], lowercase=True,
+                  interval_metadata=exp[2])
+
+        self.assertEqual(exp, obs)
+
+    def test_embl_to_protein(self):
+        # TODO: add protein support
+        i = 0
+        # there is no support for protein at the moment
+#        exp = self.multi[i]
```

Comment:
can the `#` marks be indented properly?

In general, we advise removing commented out code as it can be misleading. However, in this case, I think it is appropriate. Would it be possible to add to the comment a mention explicitly stating that the following code should be a valid test once protein support is added?